ui/src/components/schemas/Schemas.tsx

```diff
@@ -47,7 +51,7 @@ export function Schemas() {
           title="Credential schemas explained"
         />
 
-        <MySchemas />
+        <SchemasTable />
```

Comment:
Currently, the `paddingLeft` and the `paddingRight` is different at each level of the application. The side padding in the header is different than the side padding of the title and the description on the page, which at the same time is different than the padding of the table. Ee should use the same padding for all the items in the screen to make it beautiful for the sight (all should be alined to the red line in this screenshot, for example).

<img width="562" alt="Captura de pantalla 2023-05-11 a las 19 59 25" src="https://github.com/0xPolygonID/issuer-node/assets/6737805/628e89fd-77dd-4cf9-82e4-e46ca40a8782">

Reply:
I'm going to respect the design that has more padding 
![image](https://github.com/0xPolygonID/issuer-node/assets/19190248/f434981d-d205-4c4b-9685-12739ccb3603)

Reply:
I've just seen that in the designs that red line is not respected neither... 😶‍🌫️

Reply:
I've pushed a change to align everything, let me know your opinion